src/view/renderer.js

```diff
@@ -447,6 +501,15 @@ export default class Renderer {
 	 */
 	_updateAttrs( viewElement ) {
 		const domElement = this.domConverter.mapViewToDom( viewElement );
+
+		if ( !domElement ) {
```

Comment:
I'm not sure about this – this function should not be called if the mapping is broken. Can't we clean `Renderer#markedAttribiutes` earlier, when updating the mappings.

In other words – if you call a function, you expect it to do something. You should be more or less sure it makes sense.

Reply:
AFAIR the main reason was that when we update bindings of a specific element (it is removed from mappings and also all its children mappings are removed). So if you would like to update `markedAttributes` during updating bindings you will have to iterate:

1. through all updated element children to check and remove them from `markedAttributes`,
2. or through `markedAttributes` and check if any elements mappings were removed.

It just seemed simpler to check it here if we anyway iterate through `markedAttributes` (and this is just an equivalent to **2nd** point just in different place). WDYT @Reinmar ?